firefox-ios/Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -70,6 +71,11 @@ class TopTabsViewController: UIViewController, Themeable, Notifiable, FeatureFla
         button.setImage(UIImage.templateImageNamed(StandardImageIdentifiers.Large.plus), for: .normal)
         button.semanticContentAttribute = .forceLeftToRight
         button.addTarget(self, action: #selector(TopTabsViewController.newTabTapped), for: .touchUpInside)
+        let longPressRecognizer = UILongPressGestureRecognizer(
```

Comment:
The long press should only be enabled when the `toolbarOneTapNewTab` feature is enabled, I think.